docs/user/tutorials/vote-extensions/mitigate-frontrunning.md

```diff
@@ -0,0 +1,500 @@
+# Mitigate Auction Front-Running
+
+In this tutorial, we will be covering:
+
+* [What are Vote extensions?](#what-are-vote-extensions)
+* [Context](#context)
+* [Simulation of Auction Front-Running](#simulation-of-auction-front-running)
+* [Mitigating Front-running with Vote Extensions](#mitigating-front-running-with-vote-extensions)
+* [Demo of Mitigating Front-Running with Vote Extensions](#demo-of-mitigating-front-running-with-vote-extensions)
+
+## What are Vote extensions?
+
+Vote extensions is arbitrary information which can be inserted into a block. This feature is part of ABCI 2.0, which is available for use in the SDK 0.50 release and part of the 0.38 CometBFT release. 
+
+More information about vote extensions can be seen [here](../../../build/abci/03-vote-extensions.md).
+
+## Context
+
+For this tutorial we are using an example of an application that is mitigating auction front-running. 
```

Comment:
Can we expand a little bit on what the example application does? Explain beforehand that the application is an auctioning of namespaces, just a couple of lines is fine.